man/corosync.conf.5

```diff
@@ -394,7 +394,11 @@ the token is retransmitted.  This will be automatically calculated if token
 is modified.  It is not recommended to alter this value without guidance from
 the corosync community.
 
-The default is 238 milliseconds.
+The minimum is 30 milliseconds if you set. If not set and error occur, make sure
```

Comment:
I don't think `if you set` is really needed. `Minimum is 30 milliseconds.` is enough.

Reply:
"if you set" also sounds very odd on English. "If set" is better if that's needed